pkg/utils/color/color.go

```diff
@@ -0,0 +1,27 @@
+package utils
+
+import "runtime"
+
+var Reset = "\033[0m"
+var Red = "\033[31m"
+var Green = "\033[32m"
+var Yellow = "\033[33m"
+var Blue = "\033[34m"
+var Purple = "\033[35m"
+var Cyan = "\033[36m"
+var Gray = "\033[37m"
+var White = "\033[97m"
```

Comment:
I suggest to use existing go pkg for terminal color, just search about "go terminal color" or something. it may prevent us from writing our own test to make the coverage to be 100% too.

Reply:
any package suggestion? or should i just dump the very idea of giving colors to user prompt 😄

Reply:
alright